numba/unicode.py

```diff
@@ -622,6 +623,34 @@ def join_str_impl(sep, parts):
         return join_str_impl
 
 
+@overload_method(types.UnicodeType, 'zfill')
+def unicode_zfill(string, width):
+    if not isinstance(width, types.Integer):
+        raise TypingError("<width> must be an Integer")
+
+    def zfill_impl(string, width):
+
+        str_len = len(string)
+
+        if width <= str_len:
+            return string
+
+        if str_len != 0:
+            first_char = string[0]
+
+        extra_zero = width - str_len
+        fill_line = '0' * extra_zero
```

Comment:
also, maybe you could shrink this into one line, since `extra_zero` is only used once. Also, can you rename `fill_line` to `padding`, i think this is a better name.